auth_token/contrib/common/auth_security/views.py

```diff
@@ -4,20 +4,22 @@
 from auth_token.contrib.common.default.views import TokenLoginView as DefaultTokenLoginView
 
 from security.decorators import throttling_all
-from security.enums import InputLoggedRequestType
+from security.enums import InputRequestSlug
 
 
 @throttling_all(*LOGIN_THROTTLING_VALIDATORS)
 class TokenLoginView(DefaultTokenLoginView):
 
     def form_valid(self, form):
-        if getattr(self.request, 'input_logged_request', False):
-            self.request.input_logged_request.type = InputLoggedRequestType.SUCCESSFUL_LOGIN_REQUEST
+        input_request_logger = getattr(self.request, 'input_request_logger', None)
```

Comment:
next security version will contain helper which remove this duplicity. But additional security incomatible changes must be merged with it (i will fix it here after that)